modules/lvms-troubleshooting-recovering-from-disk-failure.adoc

```diff
@@ -6,7 +6,27 @@
 [id="recovering-from-disk-failure_{context}"]
 = Recovering from disk failure
 
-If you see a failure message while inspecting the events associated with the persistent volume claim (PVC), there might be a problem with the underlying volume or disk. Disk and volume provisioning issues often result with a generic error first, such as `Failed to provision volume with StorageClass <storage_class_name>`. A second, more specific error message usually follows.
+If you see a failure message while inspecting the events associated with the persistent volume claim (PVC), there can be a problem with the underlying volume or disk. 
+
+Disk and volume provisioning issues result with a generic error message such as `Failed to provision volume with storage class <storage_class_name>`. The generic error message is followed by a specific volume failure error message.
+
+The following table describes the volume failure error messages:
+
+.Volume failure error messages
+[%autowidth, options="header"]
+|===
+
+|Error message |Description
+|`Failed to check volume existence` |Indicates a problem in verifying whether the volume already exists. Volume verification failure can be caused by network connectivity problems or other failures.
+|`Failed to bind volume` |Failure to bind a volume can happen if the persistent volume (PV) that is available does not match the requirements of the PVC.
+|`FailedMount or FailedUnMount` |This error indicates problems when trying to mount the volume to a node or unmount a volume from a node. If the disk has failed, this error can appear when a pod tries to use the PVC.
```

Comment:
Is this one error message with an "or" in it, or is it two possible messages (like so:)?
```suggestion
|`FailedMount` or `FailedUnMount` |This error indicates problems when trying to mount the volume to a node or unmount a volume from a node. If the disk has failed, this error can appear when a pod tries to use the PVC.
```